package-system/glad-multiplatform/Findglad.cmake

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(MY_NAME "glad")
```

Comment:
Glad in 3rd party is used by legacy only. Previously when moving 3rd party, legacy wasn't fully deprecated. But now, I think we don't need this anymore.
Atom has its own Glad, so not having this one won't affect Atom.